app/models/type/attributes.rb

```diff
@@ -32,13 +32,15 @@ module Type::Attributes
   EXCLUDED = %w[_type
                 _dependencies
                 attribute_groups
-                links parent_id
+                links
+                parent_id
                 parent
                 description
                 schedule_manually
                 derived_start_date
                 derived_due_date
-                derived_estimated_time].freeze
+                derived_estimated_time
+                duration].freeze
```

Comment:
@dombesz apparently `duration` isn't the first property to be excluded from the form configuration. So adding it here got rid of the property.